### ISSUE TITLE
playMath's seed is deterministic but gets 100% test acc on my machine

### ISSUE THREAD

**Issue by jaymefosa** [karpathy/minGPT#34]
![image](https://user-images.githubusercontent.com/92468/91411109-bef93780-e7fc-11ea-99a5-ef54cf76a393.png)

torch 1.5.0

**Comment by fpgaminer** [comment]
The model and training code has seen a number of updates, some of which fixed bugs that improved training, but looking at the history for `play_math` it hasn't been updated since the first commit.  So I imagine the results you got are up-to-date results, and the ones shown in the current commit of `play_math` are out of date.
